catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java

```diff
@@ -565,11 +548,10 @@ private StringBuilder appendColumnDefinition(JdbcColumn column, StringBuilder sq
       sqlBuilder.append("DEFAULT '").append(column.getDefaultValue()).append("'").append(SPACE);
     }
 
-    // Add column properties if specified
-    if (CollectionUtils.isNotEmpty(column.getProperties())) {
-      for (String property : column.getProperties()) {
-        sqlBuilder.append(property).append(SPACE);
-      }
+    // Add column auto-increment
+    if (column.autoIncrement()) {
```

Comment:
I prefer to support column.autoIncrement in Mysql, because it's straight forward